model/implementation/DeliveryService.php

```diff
@@ -36,7 +41,19 @@
 class DeliveryService extends ConfigurableService
     implements ProctorAssignment
 {
+    /**
+     * QtiSm AssessmentTestSession Storage Service
+     * @var AbstractStorage
+     */
+    private $storage;
+
+    /**
+     * temporary variable until proper servicemanager integration
+     * @var ExtendedStateService
+     */
+    private $extendedStateService;
 
+    const STATE_ACTIVE = 'ACTIVE';
```

Comment:
as discussed, would suggest const STATE_INIT = 'INIT'; since it better matches better with what it represents in the proctoring delivery context : the only way to be in this state is when the delivery is initialized